docs/getting-started/about.rst

```diff
@@ -1,3 +1,6 @@
 ===============
 About PyMedPhys
-===============
\ No newline at end of file
+===============
+
+PyMedPhys is a toolbox for Medical Physicists that we all have access to, and
+we will all have access to whatever it becomes in the future.
```

Comment:
Sorry, I know you asked me to draft something. I found it very difficult to start (and not simply regurgitate the README, so I left it for further thought/discussion. :)

This is a perfectly suitable start!

Reply:
In the end I imported just the description and team from readme :)

Reply:
No need to be sorry :)